model_compression_toolkit/qat/pytorch/quantizer/quantization_dispatcher_builder.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2022 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+
+from typing import Dict
+from model_compression_toolkit.core import common
+from model_compression_toolkit.core.common.framework_info import FrameworkInfo
+from model_compression_toolkit import qunatizers_infrastructure as qi
+from model_compression_toolkit.qat.pytorch.quantizer.ste_rounding.symmetric_ste import STEWeightQuantizer
+from model_compression_toolkit.qat.pytorch.quantizer.ste_rounding.uniform_ste import STEUniformWeightQuantizer
+
+METHOD2QUANTIZER = {qi.QuantizationMethod.SYMMETRIC: STEWeightQuantizer,
+                    qi.QuantizationMethod.POWER_OF_TWO: STEWeightQuantizer,
+                    qi.QuantizationMethod.UNIFORM: STEUniformWeightQuantizer}
+
+
+def quantization_dispatcher_builder(n: common.BaseNode,
+                                    fw_info: FrameworkInfo,
+                                    method2quantizer: Dict[
+                                        qi.QuantizationMethod, qi.BasePytorchQuantizer] = METHOD2QUANTIZER) -> qi.PytorchNodeQuantizationDispatcher:
```

Comment:
Is it possible to use DefaultDict for default argument? It's not a good practice to define a mutable variable as a default value for an argument (See use of [DefaultDict ](https://github.com/sony/model_optimization/blob/38d21ce503b4257b3ecc113281e8f2bca8cebc87/model_compression_toolkit/gptq/keras/quantizer/gumbel_rounding/base_gumbel_rounding.py#L85)) as a default value.

Alternatively (which I think is better), set the default value to None, and do:
`if method2quantizer is None: method2quantizer  = METHOD2QUANTIZER `